internal/ethapi/api.go

```diff
@@ -1722,10 +1725,11 @@ func AccessList(ctx context.Context, b CeloBackend, blockNrOrHash rpc.BlockNumbe
 
 		exchangeRates := emptyExchangeRates
 		if args.FeeCurrency != nil {
-			// Always use the header's parent here, since we want to create the list at the
-			// queried block, but want to use the exchange rates before (at the beginning of)
-			// the queried block
-			exchangeRates, err = b.GetExchangeRates(ctx, header.ParentHash)
+			feeBlockNum, err := rpc.BlockNumberOrHashEnsureHashOnly(ctx, b, blockNrOrHash, true, true)
```

Comment:
It would be good to carry over the comment, especially in how it related to the last two parameters here (and in the call above).